e2e/cypress/tests/integration/bot_accounts/tags_spec.js

```diff
@@ -48,7 +48,9 @@ describe('Bot tags', () => {
                 await client.pinPost(postId);
 
                 cy.visit(`/${team.name}/channels/${channel.name}`);
-                cy.clickPostDotMenu(postId);
```

Comment:
can we update the `clickPostDotMenu` itself?

Reply:
> can we update the `clickPostDotMenu` itself?

Thanks @M-ZubairAhmed, for the review,  `clickPostDotMenu` function is still required for features or other test cases. These changes are only to trigger the emoji menu using mouseover